src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -217,27 +218,28 @@ function snow_liquid_mass_fraction(
     SWE::FT,
     parameters::SnowParameters{FT},
 ) where {FT}
-    _ρ_i = FT(LP.ρ_cloud_ice(parameters.earth_param_set))
+    #if SWE < eps(FT) return FT(1) end #do this to speed up computation or not, compared to the max() in the final line?
+    #_ρ_i = FT(LP.ρ_cloud_ice(parameters.earth_param_set)) #why do we define this if we never use it?
     _ρ_l = FT(LP.ρ_cloud_liq(parameters.earth_param_set))
     _T_ref = FT(LP.T_0(parameters.earth_param_set))
     _T_freeze = FT(LP.T_freeze(parameters.earth_param_set))
     _LH_f0 = FT(LP.LH_f0(parameters.earth_param_set))
     _cp_i = FT(LP.cp_i(parameters.earth_param_set))
     _cp_l = FT(LP.cp_l(parameters.earth_param_set))
 
-    ΔT = _T_freeze - _T_ref
+    #ΔT = _T_freeze - _T_ref #why do we define this if we never use it?
 
     _ρcD_g = parameters.ρcD_g
     Uminus =
         (_ρ_l * SWE * _cp_i + _ρcD_g) * (_T_freeze - _T_ref) -
         _ρ_l * SWE * _LH_f0
     Uplus = (_ρ_l * SWE * _cp_l + _ρcD_g) * (_T_freeze - _T_ref)
     if U < Uminus
-        FT(0)
```

Comment:
this returns it, but maybe you think it is clearer to have the return statement?

Reply:
I thought it felt more consistent with the rest of the functions but I agree the change isn't really "changing" anything